### PR TITLE
RELEASE: 1.2.0

### LINKED ISSUE #60: Add the file titles to all XML files
ArcGIS Pro does not automatically populate the Title field of the metadata. It would be great if we could do that based on the filename being used to generate the xml file.

### PR DESCRIPTION
**Pull request by Jlrine2**
### Added
* Readme file for InSAR products
* `InSarMetadataWriter` class to handle generation of InSar metadata files

### Changed
* Provided different attribution statement options for different uses
* Updated link for new version of the RTC Product Guide

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,28 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Added
+* Readme file for InSAR products
+* `InSarMetadataWriter` class to handle generation of InSar metadata files
+
+### Changed
+* Provided different attribution statement options for different uses
+* Updated link for new version of the RTC Product Guide 
+
+## [1.0.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v0.4.2...v1.0.0)
```
Comment (Jlrine2) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## [1.2.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v0.4.2...v1.0.0)
+
+### Added
+* Readme file for InSAR products
+* `InSarMetadataWriter` class to handle generation of InSar metadata files
+
+### Changed
+* Provided different attribution statement options for different uses
+* Updated link for new version of the RTC Product Guide 
+
```
Comment (Jlrine2) [inline CHANGELOG.md:19]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,14 +6,26 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [1.2.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v0.4.2...v1.0.0)
```
Comment (asjohnston-asf) [inline CHANGELOG.md]:
```suggestion
## [1.2.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v1.1.0...v1.2.0)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,14 +6,26 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Changed
+* Provided different attribution statement options for different uses
+* Updated link for new version of the RTC Product Guide 
+
 ## [1.1.0]
 ### Changed
 - Added DB option to RTC Readme
 
 ## [1.0.0]
```
Comment (asjohnston-asf) [inline CHANGELOG.md]:
```suggestion
## [1.0.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v0.4.2...v1.0.0)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,14 +6,26 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Added
+* Readme file for InSAR products
+* `InSarMetadataWriter` class to handle generation of InSar metadata files
+
+### Changed
+* Provided different attribution statement options for different uses
+* Updated link for new version of the RTC Product Guide 
+
 ## [1.1.0]
```
Comment (asjohnston-asf) [inline CHANGELOG.md]:
```suggestion
## [1.1.0](https://github.com/ASFHyP3/hyp3-metadata-templates/compare/v1.0.0...v1.1.0)
```

File: hyp3_metadata/templates/insar/inc_map_tif.xml.j2 [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+{% endblock %}
+{% block dataLineage %}
+The incidence angle map is generated from the DEM and metadata from the reference scene using GAMMA's GC_map_mod program.
+
+For a detailed description of InSAR processing using GAMMA software, refer to the readme file included with the product package.
+{% endblock %}
+
+******************
+
+{{ Begin idAbs READABLE - not to be included in the xml template }}
```
Comment (asjohnston-asf) [inline hyp3_metadata/templates/insar/inc_map_tif.xml.j2]: should this be commented like in the other templates?

Comment (Jlrine2) [inline hyp3_metadata/templates/insar/inc_map_tif.xml.j2]:
```suggestion
{# Begin idAbs READABLE - not to be included in the xml template
```

File: hyp3_metadata/templates/insar/inc_map_tif.xml.j2 [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Refer to ASF's [InSAR Product Guide](https://hyp3-docs.asf.alaska.edu/guides/insar_product_guide/) for more information about On Demand InSAR products.
+
+The Sentinel-1A satellite was launched April 3, 2014, and the Sentinel-1B satellite was launched April 25, 2016. The satellites each have a 12-day repeat cycle. More information about the mission is available at https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1
+
+Additional information about Sentinel-1 data, imagery, tools and applications is available at https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/
+{{ End idAbs READABLE - not to be included in the xml template }}
```
Comment (Jlrine2) [inline hyp3_metadata/templates/insar/inc_map_tif.xml.j2]:
```suggestion
End idAbs READABLE - not to be included in the xml template
#}
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**